src/python/espressomd/analyze.pyx

```diff
@@ -1054,15 +1056,30 @@ class Analysis:
     #
 
     def angular_momentum(self, p_type=None):
+        """
+        Calculates the system's angular momentum with respect to the origin.
+    
+        Note that virtual sites are not included, as they do not have a meaningful mass.
+    
+        Parameters
+        ----------
+        p_type : :obj:`int`
+            Particle :attr:`~espressomd.particle_data.ParticleHandle.type` for
+            which to calculate the center of mass.
+    
+        Returns
+        -------
+        array of :obj:`float`
```

Comment:
```(3,) array_like of :obj:`float` ```

Reply:
It returns an ndarray. Do you still think, it should be array_like?

Reply:
yes, why not?

Reply:
Do we want to state a warning in the description of the function that the angular momentum is not conserved when PBCs are applied? See e.g. https://arxiv.org/abs/1312.7008